core/src/main/java/org/apache/accumulo/fate/util/Retry.java

```diff
@@ -133,8 +156,8 @@ public void useRetry() {
     if (!canRetry()) {
       throw new IllegalStateException("No retries left");
     }
-
     retriesDone++;
+    firstRetry = false;
```

Comment:
This is redundant. retriesDone already tells us whether we've done any retries or not.